lib/src/login/login_provider_list.dart

```diff
@@ -94,10 +97,10 @@ class _ProviderListState extends State<ProviderList> {
   @override
   Widget build(BuildContext context) {
     if (widget.type == ProviderListType.login) {
-      title = L10n.get(L.loginSignIn);
+      title = ProviderList.loginViewTitle;
       text = L10n.get(L.loginChooseProvider);
     } else if (widget.type == ProviderListType.register) {
-      title = L10n.get(L.register);
+      title = ProviderList.registerViewTitle;
```

Comment:
How about:
```
static get viewTitle {
  return type == ProviderListType.login ? L10n.get(L.loginSignIn) : L10n.get(L.register);
}
static get viewText {
   return type == ProviderListType.login ? L10n.get(L.loginChooseProvider) : L10n.get(L.providerRegisterChoose);
}
```
So, this way we will have just a `viewTitle` and a `viewText`, and we can get rid of `String title` and `String text`.

Reply:
This is not possible because the instance member `type `can't be accessed from a static method.